jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java

```diff
@@ -1,34 +1,125 @@
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Winter");
+        radioButtonSection.springNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Spring");
+        radioButtonSection.summerNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Summer");
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Autumn");
+    }
+
+
+    @Test
+    public void onlyOneWinterButtonEnabledTest() {
+        radioButtonSection.winterNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
+        radioButtonSection.springNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", disableSeasonButton);
+    }
+
     @Test
-    public void basicTest() {
-        basicRadioButtons.is().displayed();
-        firstBasicRadioButton.is().displayed();
-        secondBasicRadioButton.is().displayed();
+    public void onlyOneSpringButtonEnabledTest() {
+        radioButtonSection.springNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.springNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", disableSeasonButton);
     }
 
     @Test
-    public void getTextTest() {
-        assertEquals(firstBasicRadioButton.getText(), "Option 1");
-        assertEquals(secondBasicRadioButton.getText(), "Option 2");
+    public void onlyOneSummerButtonEnabledTest() {
+        radioButtonSection.summerNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class",disableSeasonButton);
+        radioButtonSection.springNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", disableSeasonButton);
     }
+
+    @Test
+    public void onlyOneAutumnButtonEnabledTest() {
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.springNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
+    }
+
 }
```

Comment:
Could you please pay attention to count of empty rows between tests, somewhere you paste two rows. Could you please, delete extra rows.